### PR TITLE
Rework more commons-lang usages

### PR DESCRIPTION
**Pull request by wborn**
* Reworks many commons-lang usages to use standard Java
* Updates all remaining commons.lang imports to commons.lang3

Related to openhab/openhab-addons#7722

### CODE REVIEW

**Comment by hmerk** [comment]
Why did you make changes to the Wemo binding, there is already a PR for removing apache.commons completely.
We only need to fix the tests, but this is not easy for me, as I did not write those.

**Comment by wborn** [comment]
That's a good thing @hmerk! I can not remember the details of all PRs. :wink:

**Comment by bobadair** [comment]
LGTM for the Lutron binding.

**Review by mhilbush — APPROVED** [review]
LGTM

Thanks for doing this!

File: bundles/org.openhab.binding.satel/src/main/java/org/openhab/binding/satel/internal/command/ControlCommand.java [hunk 1]
```diff
@@ -14,7 +14,6 @@
 
 import org.eclipse.jdt.annotation.NonNullByDefault;
 import org.openhab.binding.satel.internal.protocol.SatelMessage;
-import org.openhab.binding.satel.internal.util.StringUtils;
```
Comment (druciak) [inline bundles/org.openhab.binding.satel/src/main/java/org/openhab/binding/satel/internal/command/ControlCommand.java:17]: This is not a common-lang import, I already removed this dependency.

File: bundles/org.openhab.binding.satel/src/main/java/org/openhab/binding/satel/internal/command/ControlCommand.java [hunk 2]
```diff
@@ -59,7 +58,7 @@ protected boolean isResponseValid(SatelMessage response) {
     }
 
     protected static byte[] userCodeToBytes(String userCode) {
-        if (StringUtils.isEmpty(userCode)) {
+        if (userCode.isEmpty()) {
```
Comment (druciak) [inline bundles/org.openhab.binding.satel/src/main/java/org/openhab/binding/satel/internal/command/ControlCommand.java:61]: This is not equivalent of previous code.

**Review by druciak — COMMENTED** [review]
Why have you done changes to Satel binding? It had already removed all commons dependencies.

**Review by druciak — APPROVED** [review]
Satel binding looks good

**Comment by wborn** [comment]
It showed up in my search for `StringUtils` usages @druciak. When bindings like Satel prevent null values and use null annotations   all those redundant null checks can be skipped and there shouldn't be much use for `isEmpty`, `isBlank` etc. provided by utility classes similar to the commons-lang `StringUtils`.

**Review by mherwege — APPROVED** [review]
LGTM for the upnpcontrol binding.

**Comment by marcelrv** [comment]
LGTM for max binding

**Review by cweitkamp — APPROVED** [review]

**Pull request merged**